backend/game/player/player.py

```diff
@@ -48,3 +55,9 @@ def has_instance_of_card(self, card: Card):
             if type(player_card) == type(card):
                 return True
         return False
+
+    def discard_all_cards(self):
+        self.cards.clear()
+
+    def lose_all_stars(self):
+        self.victory_points = constants.DEATH_HIT_POINT
```

Comment:
stars and hitpoints are not the same thing. I don't think DEATH_HIT_POINT should be used as a constant in lose_all_stars. I also think a constant for the integer 0 is unnecessary. I'd rather this just be 0